lib/ansible/modules/cloud/hcloud/hcloud_network.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: hcloud_network
+
+short_description: Create and manage cloud Networks on the Hetzner Cloud.
```

Comment:
```suggestion
short_description: Create and manage cloud networks on the Hetzner Cloud.
```
Is there a reason why you write `Networks` with a capital `n`?

Reply:
The product ist called `Networks` with a capital n.

Reply:
Ok, in that case it's fine :)